changelog/index.md

```diff
@@ -14,6 +14,10 @@ icon:
 
 - Release _candidates_ begin appearing on the **second Wednesday** of each month
   (Coder may create multiple release candidates prior to the actual release)
+
+> We do not provide documentation for release candidates, and they should not be
+> used unless you've been asked to use it by Coder.
```

Comment:
```suggestion
> used unless you've been instructed to use them by Coder.
```

Reply:
"unless the Coder team instructs you to do so" or something like that?